includes/wccom-site/class-wc-wccom-site.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * WooCommerce.com Product Installation.
+ *
+ * @class    WC_WCCOM_Site_Installer
+ * @package  WooCommerce/WCCOM_Site
+ * @since    3.7.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_WCCOM_Site Class
+ *
+ * Main class for WooCommerce.com connected site.
+ */
+class WC_WCCOM_Site {
+	/**
+	 * Load the WCCOM site class.
+	 *
+	 * @since 3.7.0
+	 */
+	public static function load() {
+		self::includes();
+
+		add_action( 'woocommerce_wccom_install_products', array( 'WC_WCCOM_Site_Installer', 'install' ) );
+		add_action( 'woocommerce_rest_api_get_rest_namespaces', array( __CLASS__, 'register_rest_namespace' ) );
+	}
+
+	/**
+	 * Include support files.
+	 *
+	 * @since 3.7.0
+	 */
+	protected static function includes() {
+		require_once WC_ABSPATH . 'includes/admin/helper/class-wc-helper.php';
+		require_once __DIR__ . '/class-wc-wccom-site-installer.php';
```

Comment:
@kloon is fine to use PHP 5.3 magic constant in here?

Reply:
Happy to switch to `WC_ABSPATH`, if you prefer. `__DIR__` is shorter, no strong argument.

Reply:
Replaced with `WC_ABSPATH` in https://github.com/woocommerce/woocommerce/pull/24123/commits/3f9bbf5a3bed66349b1af2e0d1ff430abe63a1ec.